shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -112,17 +113,16 @@ new PlatformMessageHandlerIos(task_runners.GetPlatformTaskRunner())) {}
 
 void PlatformViewIOS::attachView() {
   FML_DCHECK(owner_controller_);
-  FML_DCHECK(owner_controller_.get().isViewLoaded)
-      << "FlutterViewController's view should be loaded "
-         "before attaching to PlatformViewIOS.";
-  auto flutter_view = static_cast<FlutterView*>(owner_controller_.get().view);
-  auto ca_layer = fml::scoped_nsobject<CALayer>{[[flutter_view layer] retain]};
+  FML_DCHECK(owner_controller_.isViewLoaded) << "FlutterViewController's view should be loaded "
+                                                "before attaching to PlatformViewIOS.";
+  FlutterView* flutter_view = static_cast<FlutterView*>(owner_controller_.view);
```

Comment:
Is this `static_cast` needed?

Reply:
Yes, unfortunately since `owner_controller_.view` returns a `UIView*`.